### ISSUE TITLE
flatten-maven-plugin has problem,I have depencies in repo

### ISSUE THREAD

**Issue by 13567436138** [spring-projects/spring-boot#18173]
```
[ERROR] Failed to execute goal org.codehaus.mojo:flatten-maven-plugin:1.1.0:flatten (flatten) on project spring-boot-parent: 1 problem was encountered while building the effective model for org.springframework.boot:spring-boot-parent:2.2.0.BUILD-SNAPSHOT
[ERROR] [FATAL] Non-resolvable parent POM for org.springframework.data:spring-data-releasetrain:Moore-RC3: org.eclipse.aether.resolution.DependencyResolutionException: Failure to find org.springframework.data.build:spring-data-build:pom:2.2.0.RC3 in http://maven.aliyun.com/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of nexus-aliyun has elapsed or updates are forced @ org.springframework.data:spring-data-releasetrain:Moore-RC3
```

**Comment by 13567436138** [comment]
in my maven repo,I can find rg.springframework.data.build:spring-data-build:pom:2.2.0.RC3

**Comment by 13567436138** [comment]
I commet the flatten-maven-plugin plugin,it does not show error now

**Comment by wilkinsona** [comment]
This is a [known problem](https://github.com/mojohaus/flatten-maven-plugin/issues/89) with the flatten plugin. Until it's fixed, you must use Maven 3.5.4 to build Spring Boot as the wrapper in the source tree does.
